From 309c57cc9d7515c3f13dfc30632963c9f17cd0ae Mon Sep 17 00:00:00 2001
From: Subramaniam Chanderashekarapuram <subramaniam.ca@ti.com>
Date: Thu, 15 Mar 2012 17:06:44 -0500
Subject: [PATCH 090/102] rpmsg: omx: remove pvr ion header include
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

PVRExportFDToIONHandles was being called directly by rpmsg omx
driver. This is no more the case and hence PVR related header
can be removed.

Change-Id: Ied23ac3708fd297f5aa4b4080e2de494b80df401
Signed-off-by: Hemant Hariyani <hemanthariyani@ti.com>
Signed-off-by: Suman Anna <s-anna@ti.com>
Signed-off-by: Subramaniam Chanderashekarapuram <subramaniam.ca@ti.com>
Signed-off-by: D. Andrei Măceș <dmaces@nd.edu>
---
 drivers/rpmsg/rpmsg_omx.c | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/drivers/rpmsg/rpmsg_omx.c b/drivers/rpmsg/rpmsg_omx.c
index a38b1e1..5d702ea0 100644
--- a/drivers/rpmsg/rpmsg_omx.c
+++ b/drivers/rpmsg/rpmsg_omx.c
@@ -119,10 +119,6 @@ static DEFINE_SPINLOCK(rpmsg_omx_services_lock);
 static LIST_HEAD(rpmsg_omx_services_list);
 
 #ifdef CONFIG_ION_OMAP
-#ifdef CONFIG_PVR_SGX
-#include "../gpu/pvr/ion.h"
-#endif
-#endif
 
 #ifdef CONFIG_MACH_TUNA
 /*
@@ -175,7 +171,7 @@ static int _rpmsg_pa_to_da(struct rpmsg_omx_instance *omx, u32 pa, u32 *da)
 }
 #endif
 
-#if defined(CONFIG_ION_OMAP) && !defined(CONFIG_MACH_TUNA)
+#if !defined(CONFIG_MACH_TUNA)
 static void _rpmsg_buffer_update_page_list(struct rpmsg_omx_instance *omx,
 					   struct rpmsg_buffer *buffer)
 {
@@ -276,6 +272,7 @@ _rpmsg_buffer_free(struct rpmsg_omx_instance *omx, struct rpmsg_buffer *buffer)
 	kfree(buffer);
 }
 #endif
+#endif
 
 static int _rpmsg_omx_buffer_lookup(struct rpmsg_omx_instance *omx,
 					long buffer, u32 *va)
-- 
2.7.4

